aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/todo-group.csproj

```diff
@@ -6,11 +6,19 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <RootNamespace>todo_group</RootNamespace>
```

Comment:
Maybe rename the csproj to `TodoGroup.csproj`?

Reply:
Also, we should update the namespace.

Reply:
yes, agreed. Will do. @Rick-Anderson could you please verify that this sample has no additional references in the documentation? This was an existing sample that I found and modified. I'm not sure whether it will break anything somewhere

Reply:
<img width="269" alt="image" src="https://user-images.githubusercontent.com/3605364/190561663-bee3ec7d-7548-4d60-bd8b-ea0912617215.png">

Nope